bluemira/builders/tf_coils.py

```diff
@@ -641,44 +508,53 @@ def __init__(
             params.z_0.value,
             params.B_0.value,
         )
-        ripple_constraint = ripple_selector.make_ripple_constraint(
+        self._ripple_constraint = ripple_selector.make_ripple_constraint(
             parameterisation, self.solver, params.TF_ripple_limit.value, rip_con_tol
         )
         self.ripple_selector = ripple_selector
 
-        constraints = [ripple_constraint]
-
-        if keep_out_zone is not None:
-            koz_points = self._make_koz_points(keep_out_zone)
-            koz_constraint = OptimisationConstraint(
-                constrain_koz,
-                f_constraint_args={
-                    "parameterisation": parameterisation,
-                    "n_shape_discr": n_koz_points,
-                    "koz_points": koz_points,
-                },
-                tolerance=koz_con_tol * np.ones(n_koz_points),
-            )
-            constraints.append(koz_constraint)
-
-        super().__init__(parameterisation, optimiser, objective, constraints)
-
     def _make_koz_points(self, keep_out_zone: BluemiraWire) -> Coordinates:
         """
         Make a set of points at which to evaluate the KOZ constraint
         """
+        # TODO: I guess I'm gonna lose this fine control?
```

Comment:
Is this still an issue? I think the new keep-out zone class has these options.

The keep-out zone stuff is really just for convenience, if finer control is needed, then we can always just write up an inequality constraint.